### ISSUE TITLE
Selection: Multi selection mode only allows one item to be selected

### ISSUE THREAD

**Issue by kristoferbrown** [OfficeDev/office-ui-fabric-react#1707]
### Bug Report

- __Package version(s)__: Current
- __Browser and OS versions__:  All browsers

#### Describe the issue:

In multi select mode, clicking a second item deselects the first item. Marquee and select all still work correctly.

#### Expected behavior:

Mutliple items should be selectable

### If applicable, please provide a codepen repro:

Repros in current website.
